.drone.yml

```diff
@@ -167,32 +316,32 @@ name: sgx-release-ubuntu-1804
 
 steps:
 - name: prepare
-  image: teaclave/teaclave-build-ubuntu-1804-sgx-2.14:0.1.2
+  image: teaclave/teaclave-build-ubuntu-1804-sgx-2.14:0.1.3
   commands:
   - . /root/.cargo/env
   - . /opt/sgxsdk/environment
   - mkdir -p build
   - cd build && cmake -DCMAKE_BUILD_TYPE=Release -DTEST_MODE=OFF ..
 - name: check
-  image: teaclave/teaclave-build-ubuntu-1804-sgx-2.14:0.1.2
+  image: teaclave/teaclave-build-ubuntu-1804-sgx-2.14:0.1.3
   commands:
   - . /root/.cargo/env
   - . /opt/sgxsdk/environment
   - cd build && make check
 - name: compile
-  image: teaclave/teaclave-build-ubuntu-1804-sgx-2.14:0.1.2
+  image: teaclave/teaclave-build-ubuntu-1804-sgx-2.14:0.1.3
```

Comment:
The repeated image values can be optimized by `Anchor`, it is a Yaml feature, so we can safely apply it to any Yaml files.

Reply:
Thanks, this is very helpful!